.github/workflows/build.yml

```diff
@@ -0,0 +1,46 @@
+name: build
+on:
+  push:
+    branches:
+      - '*'
+  pull_request: {}
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version-file: 'go.mod'
+      - run: "go vet ./..."
+      - name: Lint
+        uses: dominikh/staticcheck-action@v1.2.0
+        with:
+          version: "2022.1.1"
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version-file: 'go.mod'
+      - run: "make test"
+  snapshot-build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-go@v3
+        with:
+          go-version-file: 'go.mod'
+      - name: Set GOLANG_VERSION
```

Comment:
I don't see the GOLANG_VERSION being used anywhere .. why set it?

Reply:
It is used as a build flag via ldflags (https://github.com/vapor-ware/synse-snmp-plugin/blob/master/Makefile#L19) and goreleaser (https://github.com/vapor-ware/synse-snmp-plugin/blob/master/.goreleaser.yml#L14) using the same mechanism